INFO: 2018-08-17 14:42:53,157:DartRunner.py:108: Logfile is /root/Dart/log/DartRunner-2018-08-17-144253.log
INFO: 2018-08-17 14:42:53,158:DartRunner.py:109: Current Process Id: 20835
INFO: 2018-08-17 14:42:53,158:DartRunner.py:170: clusterList = dartexe 
INFO: 2018-08-17 14:42:53,158:DartRunner.py:178: dartexe
INFO: 2018-08-17 14:42:54,431:DartRunner.py:2008: NAME      READY     STATUS    RESTARTS   AGE
consul    1/1       Running   0          20h

INFO: 2018-08-17 14:42:54,545:DartRunner.py:2037: consul
INFO: 2018-08-17 14:42:54,546:DartRunner.py:2077: []
INFO: 2018-08-17 14:42:54,546:DartRunner.py:2078: []
INFO: 2018-08-17 14:42:54,546:DartRunner.py:247: /root/Dart/testset/OneTest.tst
INFO: 2018-08-17 14:42:54,546:DartRunner.py:258: Scanning File: /root/Dart/testset/OneTest.tst 
INFO: 2018-08-17 14:42:54,556:DartRunner.py:273: Reading tempRunFile : /tmp/OneTest.tst
INFO: 2018-08-17 14:42:54,792:DartRunner.py:2154: Ping on cluster dartexe Successful!
INFO: 2018-08-17 14:42:54,954:DartRunner.py:523: Dart DashBoard URL: http://dartdashboard.labs.teradata.com/report?run_id=180817-116
INFO: 2018-08-17 14:42:54,966:DartRunner.py:1696: Done inserting the information of run 180817-116
INFO: 2018-08-17 14:42:54,976:DartRunner.py:2665: DB Insert Success: NullTest
INFO: 2018-08-17 14:42:56,478:DartRunner.py:754: Looping the Main Loop: 1
INFO: 2018-08-17 14:42:56,582:DartRunner.py:1916: Running CheckHealth on cluster dartexe
INFO: 2018-08-17 14:42:58,582:DartRunner.py:941: Running Setup for the TestSet
INFO: 2018-08-17 14:42:58,583:DartRunner.py:964: {'dartexe': 'busy'}
INFO: 2018-08-17 14:42:58,583:DartRunner.py:965: {'dartexe': u'NullTest'}
INFO: 2018-08-17 14:42:58,583:DartRunner.py:975: 
Running Test: NullTest on Cluster: dartexe Thread Count: 1  

INFO: 2018-08-17 14:42:58,584:DartRunner.py:1130: Tests may be currently running..Sleeping for 10 seconds before checking the Queue!
INFO: 2018-08-17 14:42:58,585:DartRunner.py:1131: {'dartexe': 'busy'}
INFO: 2018-08-17 14:42:58,597:DartRunner.py:2555: Updated cluster: dartexe for test NullTest
ERROR: 2018-08-17 14:42:58,598:DartRunner.py:2918: Unable to run the command cat /home/beehive/bin/.build | grep 'BUILD_VERSION'
ERROR: 2018-08-17 14:42:58,598:DartRunner.py:2919: (<type 'exceptions.IndexError'>, IndexError('list index out of range',), <traceback object at 0x7fce014e4b90>)
ERROR: 2018-08-17 14:42:58,599:DartRunner.py:2920: Traceback (most recent call last):
  File "DartRunner.py", line 2908, in runCommandOnQueen
    queenNode = queenNodes[0]
IndexError: list index out of range

ERROR: 2018-08-17 14:42:58,599:DartRunner.py:2918: Unable to run the command hadoop version
ERROR: 2018-08-17 14:42:58,599:DartRunner.py:2919: (<type 'exceptions.IndexError'>, IndexError('list index out of range',), <traceback object at 0x7fce014e4b90>)
ERROR: 2018-08-17 14:42:58,599:DartRunner.py:2920: Traceback (most recent call last):
  File "DartRunner.py", line 2908, in runCommandOnQueen
    queenNode = queenNodes[0]
IndexError: list index out of range

INFO: 2018-08-17 14:42:58,599:DartRunner.py:2873: getHadoopVersion stdout = None
INFO: 2018-08-17 14:42:58,600:DartRunner.py:2874: getHadoopVersion stderr = None
INFO: 2018-08-17 14:42:58,600:DartRunner.py:2875: getHadoopVersion status = 1
INFO: 2018-08-17 14:42:58,600:DartRunner.py:2884: hdversion = None
INFO: 2018-08-17 14:42:58,600:DartRunner.py:2846: The cluster is without Hadoop, change db_version to Aster
INFO: 2018-08-17 14:42:58,610:DartRunner.py:2587: Updated Status: On test NullTest
INFO: 2018-08-17 14:42:58,610:DartRunner.py:2588: {'db_version': 'Aster', 'db_type': 'Classic', 'build_info': ''}
INFO: 2018-08-17 14:42:58,610:DartRunner.py:2214: Updating the status to Running in the database: NullTest!
INFO: 2018-08-17 14:42:58,621:DartRunner.py:2587: Updated Status: On test NullTest
INFO: 2018-08-17 14:42:58,621:DartRunner.py:2588: {'status': 'RUNNING', 'start_time': '2018-08-17 14:42:58'}
INFO: 2018-08-17 14:42:58,622:Dart.py:120: Logfile is /root/Dart/log/Dart-NullTest-180817-116-144258.log
INFO: 2018-08-17 14:42:59,148:Dart.py:390: NAME      READY     STATUS    RESTARTS   AGE
consul    1/1       Running   0          20h

INFO: 2018-08-17 14:42:59,149:Dart.py:418: consul
INFO: 2018-08-17 14:42:59,149:Dart.py:460: []
INFO: 2018-08-17 14:42:59,149:Dart.py:461: []
INFO: 2018-08-17 14:42:59,149:Dart.py:299: Test Set: SqlTests 
INFO: 2018-08-17 14:42:59,150:Dart.py:313: Running Testset: SqlTests
INFO: 2018-08-17 14:42:59,150:Dart.py:319:         Running Test: NullTest on Cluster dartexe
INFO: 2018-08-17 14:42:59,150:Dart.py:322: [u'NullTest', {u'OWNER': u'divya.sivanandan', 'testSet': u'SqlTests', 'testPlan': None, 'tester': 'root', 'projectName': 'MLEngine', 'buildNumber': '', 'buildName': 'beehive-ci-pipeline-main', u'USER': u'beehive', u'TIMEOUT': 800, u'PASSWORD': u'beehive', 'id': 1518329, u'CATEGORY': u'positive', u'EXPECTEDOUT': u'testsrc/queen/sql/test_sql/expectedQueryResults/null.out', u'NAME': u'NullTest', 'timeScale': 1, u'DATABASE': u'beehive', 'reTryCnt': 0, u'COMPONENT': u'queen/sql/test_sql', u'PRIORITY': u'2', 'runId': u'180817-116', u'TEMPLATE': u'True', u'ACTPARAMS': u'-a', 'release': 'GGR3', u'QUERY': u'testsrc/queen/sql/test_sql/input/null.sql', 'branchName': '', u'TYPE': u'SQL', 'testId': 1}, 0]
INFO: 2018-08-17 14:42:59,230:Dart.py:752: --------------------------------------------------
INFO: 2018-08-17 14:42:59,230:TestBase.py:42:           Using the following Test Parameters:
INFO: 2018-08-17 14:42:59,230:TestBase.py:43: --------------------------------------------------
INFO: 2018-08-17 14:42:59,231:TestBase.py:45:           OWNER : divya.sivanandan 
INFO: 2018-08-17 14:42:59,231:TestBase.py:45:           testSet : SqlTests 
INFO: 2018-08-17 14:42:59,231:TestBase.py:45:           testPlan : None 
INFO: 2018-08-17 14:42:59,231:TestBase.py:45:           tester : root 
INFO: 2018-08-17 14:42:59,231:TestBase.py:45:           projectName : MLEngine 
INFO: 2018-08-17 14:42:59,231:TestBase.py:45:           logPath : /root/Dart 
INFO: 2018-08-17 14:42:59,232:TestBase.py:45:           buildNumber :  
INFO: 2018-08-17 14:42:59,232:TestBase.py:45:           buildName : beehive-ci-pipeline-main 
INFO: 2018-08-17 14:42:59,232:TestBase.py:45:           USER : beehive 
INFO: 2018-08-17 14:42:59,232:TestBase.py:45:           TIMEOUT : 800 
INFO: 2018-08-17 14:42:59,232:TestBase.py:45:           filePath : queen/sql/test_sql 
INFO: 2018-08-17 14:42:59,232:TestBase.py:45:           PASSWORD : beehive 
INFO: 2018-08-17 14:42:59,232:TestBase.py:45:           id : 1518329 
INFO: 2018-08-17 14:42:59,233:TestBase.py:45:           CATEGORY : positive 
INFO: 2018-08-17 14:42:59,233:TestBase.py:45:           EXPECTEDOUT : testsrc/queen/sql/test_sql/expectedQueryResults/null.out 
INFO: 2018-08-17 14:42:59,233:TestBase.py:45:           NAME : NullTest 
INFO: 2018-08-17 14:42:59,233:TestBase.py:45:           timeScale : 1 
INFO: 2018-08-17 14:42:59,233:TestBase.py:45:           DATABASE : beehive 
INFO: 2018-08-17 14:42:59,233:TestBase.py:45:           reTryCnt : 0 
INFO: 2018-08-17 14:42:59,233:TestBase.py:45:           COMPONENT : queen/sql/test_sql 
INFO: 2018-08-17 14:42:59,234:TestBase.py:45:           PRIORITY : 2 
INFO: 2018-08-17 14:42:59,234:TestBase.py:45:           logTimestamp : 2018-08-17-144259 
INFO: 2018-08-17 14:42:59,234:TestBase.py:45:           runId : 180817-116 
INFO: 2018-08-17 14:42:59,234:TestBase.py:45:           TEMPLATE : True 
INFO: 2018-08-17 14:42:59,234:TestBase.py:45:           ACTPARAMS : -a 
INFO: 2018-08-17 14:42:59,234:TestBase.py:45:           release : GGR3 
INFO: 2018-08-17 14:42:59,235:TestBase.py:45:           QUERY : testsrc/queen/sql/test_sql/input/null.sql 
INFO: 2018-08-17 14:42:59,235:TestBase.py:45:           branchName :  
INFO: 2018-08-17 14:42:59,235:TestBase.py:45:           TYPE : SQL 
INFO: 2018-08-17 14:42:59,235:TestBase.py:45:           testId : 1 
INFO: 2018-08-17 14:42:59,235:TestBase.py:46: --------------------------------------------------
INFO: 2018-08-17 14:42:59,247:Kubernetes.py:82: Executing command [10.25.80.32]: kubectl get pods --namespace=cloud-aster
INFO: 2018-08-17 14:42:59,775:TestBase.py:170: NAME      READY     STATUS    RESTARTS   AGE
consul    1/1       Running   0          20h

INFO: 2018-08-17 14:42:59,776:Kubernetes.py:82: Executing command [10.25.80.32]: kubectl get svc queenexec-service -o json jsonpath={.spec.Ports} -n cloud-aster
ERROR: 2018-08-17 14:43:00,222:Dart.py:772: list index out of range
ERROR: 2018-08-17 14:43:00,222:Dart.py:773: The test raised an Unexpected Exception!
ERROR: 2018-08-17 14:43:00,222:Dart.py:774: (<type 'exceptions.IndexError'>, IndexError('list index out of range',), <traceback object at 0x7fce009e8050>)
ERROR: 2018-08-17 14:43:00,248:Dart.py:775: Traceback (most recent call last):
  File "/root/Dart/Dart.py", line 758, in runTest
    result = testInstance.run()
  File "/root/Dart/lib/SqlRun.py", line 61, in run
    actParams, diffParams)
  File "/root/Dart/lib/TestBase.py", line 545, in excSqlFileAndCompareOutput
    queenNode = self.queenNames[0]
IndexError: list index out of range

INFO: 2018-08-17 14:43:00,249:Dart.py:781: The test NullTest has FAILED
ERROR: 2018-08-17 14:43:00,249:Dart.py:325: list index out of range
ERROR: 2018-08-17 14:43:00,250:Dart.py:327: (<type 'exceptions.IndexError'>, IndexError('list index out of range',), <traceback object at 0x7fce001ac0e0>)
ERROR: 2018-08-17 14:43:00,250:Dart.py:328: Traceback (most recent call last):
  File "/root/Dart/Dart.py", line 323, in __init__
    self.runTest(testset, test[0], test[1], test[2])
  File "/root/Dart/Dart.py", line 758, in runTest
    result = testInstance.run()
  File "/root/Dart/lib/SqlRun.py", line 61, in run
    actParams, diffParams)
  File "/root/Dart/lib/TestBase.py", line 545, in excSqlFileAndCompareOutput
    queenNode = self.queenNames[0]
IndexError: list index out of range

INFO: 2018-08-17 14:43:00,250:Dart.py:329: Continuing with the remaining tests!
INFO: 2018-08-17 14:43:00,251:Dart.py:334: Test Set : SqlTests Total Execution Time: 1 
INFO: 2018-08-17 14:43:00,262:DartRunner.py:2587: Updated Status: On test NullTest
INFO: 2018-08-17 14:43:00,262:DartRunner.py:2588: {'status': 'FAIL', 'execution_time': '00:00:01', 'log_location': u'/root/DartLogs/dart-main-sls12-05_NullTest_2018-08-17-144259.log', 'end_time': '2018-08-17 14:43:00'}
INFO: 2018-08-17 14:43:00,624:DartRunner.py:2647: Successfully uploaded the Dart Test Log: dart-main-sls12-05_NullTest_2018-08-17-144259.log
INFO: 2018-08-17 14:43:00,633:DartDBConnect.py:127: Running the sql:
INFO: 2018-08-17 14:43:00,634:DartDBConnect.py:128: SELECT retry_logs from darttest WHERE id=1518329;
INFO: 2018-08-17 14:43:00,635:DartDBConnect.py:132: Select Return Value ========
INFO: 2018-08-17 14:43:00,635:DartDBConnect.py:133: [(None,)]
INFO: 2018-08-17 14:43:00,636:DartRunner.py:2629: Select Success: (None,) test NullTest
INFO: 2018-08-17 14:43:00,636:DartRunner.py:2630: [(None,)]
INFO: 2018-08-17 14:43:00,636:DartRunner.py:2332: RETRY LOG STRING =====
INFO: 2018-08-17 14:43:00,636:DartRunner.py:2333: None
INFO: 2018-08-17 14:43:00,649:DartRunner.py:2587: Updated Status: On test NullTest
INFO: 2018-08-17 14:43:00,649:DartRunner.py:2588: {'status': 'FAIL', 'execution_time': '00:00:01', 'log_location': u'/root/DartLogs/dart-main-sls12-05_NullTest_2018-08-17-144259.log', 'retry_count': 0, 'end_time': '2018-08-17 14:43:00', 'retry_logs': u'/root/DartLogs/dart-main-sls12-05_NullTest_2018-08-17-144259.log'}
INFO: 2018-08-17 14:43:00,649:DartRunner.py:2727: Attempting to get the Kernel and os information from the Cluster
ERROR: 2018-08-17 14:43:00,649:DartRunner.py:2918: Unable to run the command lsb_release -a
ERROR: 2018-08-17 14:43:00,650:DartRunner.py:2919: (<type 'exceptions.IndexError'>, IndexError('list index out of range',), <traceback object at 0x7fce009d6ea8>)
ERROR: 2018-08-17 14:43:00,650:DartRunner.py:2920: Traceback (most recent call last):
  File "DartRunner.py", line 2908, in runCommandOnQueen
    queenNode = queenNodes[0]
IndexError: list index out of range

ERROR: 2018-08-17 14:43:00,650:DartRunner.py:2716: Can't get kernel or os information from cluster
ERROR: 2018-08-17 14:43:00,650:DartRunner.py:2918: Unable to run the command cat /home/beehive/bin/.build | grep 'BUILD_VERSION'
ERROR: 2018-08-17 14:43:00,650:DartRunner.py:2919: (<type 'exceptions.IndexError'>, IndexError('list index out of range',), <traceback object at 0x7fce009ce3f8>)
ERROR: 2018-08-17 14:43:00,651:DartRunner.py:2920: Traceback (most recent call last):
  File "DartRunner.py", line 2908, in runCommandOnQueen
    queenNode = queenNodes[0]
IndexError: list index out of range

INFO: 2018-08-17 14:43:00,660:DartRunner.py:2587: Updated Status: On test NullTest
INFO: 2018-08-17 14:43:00,661:DartRunner.py:2588: {'revision': ''}
INFO: 2018-08-17 14:43:08,595:DartRunner.py:1134: Done with initiating all the tests!
INFO: 2018-08-17 14:43:08,595:DartRunner.py:1209: Some tests are still running! Waiting for 10 seconds!
INFO: 2018-08-17 14:43:18,601:DartRunner.py:1213: Return Value:
INFO: 2018-08-17 14:43:18,601:DartRunner.py:2958:        SqlTests       NullTest     False         1 2018-08-17 14:42:59 2018-08-17 14:43:00
INFO: 2018-08-17 14:43:18,601:DartRunner.py:1647: -------------------------------------------------------------------------------------------------
INFO: 2018-08-17 14:43:18,602:DartRunner.py:1648: returnValue = {'test': OrderedDict([(u'NAME', u'NullTest'), (u'TYPE', u'SQL'), (u'QUERY', u'testsrc/queen/sql/test_sql/input/null.sql'), (u'EXPECTEDOUT', u'testsrc/queen/sql/test_sql/expectedQueryResults/null.out'), (u'TIMEOUT', 800), (u'PRIORITY', u'2'), (u'TEMPLATE', u'True'), (u'OWNER', u'divya.sivanandan'), (u'CATEGORY', u'positive'), (u'COMPONENT', u'queen/sql/test_sql'), (u'USER', u'beehive'), (u'PASSWORD', u'beehive'), (u'DATABASE', u'beehive'), (u'ACTPARAMS', u'-a'), ('runId', u'180817-116'), ('testSet', u'SqlTests'), ('testId', 1), ('timeScale', 1), ('projectName', 'MLEngine'), ('testPlan', None), ('buildName', 'beehive-ci-pipeline-main'), ('buildNumber', ''), ('tester', 'root'), ('release', 'GGR3'), ('branchName', ''), ('reTryCnt', 0), ('id', 1518329)]), 'clusterName': 'dartexe', 'totalTime': 1, 'testName': u'NullTest', 'testResults': OrderedDict([(u'SqlTests', [[u'NullTest', 1, False, '2018-08-17 14:42:59', '2018-08-17 14:43:00', u'/root/Dart/log/queen/sql/test_sql/dart-main-sls12-05_NullTest_2018-08-17-144259.log', u'dart-main-sls12-05_NullTest_2018-08-17-144259.log']])])}
INFO: 2018-08-17 14:43:18,602:DartRunner.py:1649: -------------------------------------------------------------------------------------------------
INFO: 2018-08-17 14:43:18,602:DartRunner.py:1219: Thread Count Went Down 0 
INFO: 2018-08-17 14:43:18,602:DartRunner.py:1237: The test NullTest returned back with the following result: False
INFO: 2018-08-17 14:43:18,841:DartRunner.py:2154: Ping on cluster dartexe Successful!
INFO: 2018-08-17 14:43:18,955:DartRunner.py:1260: Freeing up the cluster dartexe
INFO: 2018-08-17 14:43:18,955:DartRunner.py:1207: All the tests have been executed!
INFO: 2018-08-17 14:43:18,976:DartRunner.py:1309: {'dartexe': 'free'}
ERROR: 2018-08-17 14:43:18,979:DartRunner.py:1643: Send email fail: /bin/sh: 14: mail: not found
sh: echo: I/O error

INFO: 2018-08-17 14:43:19,346:DartRunner.py:2962: DartRunner Final Results!
INFO: 2018-08-17 14:43:19,346:DartRunner.py:2963:         TESTSET       TESTNAME    STATUS TOTALTIME           STARTTIME             ENDTIME
INFO: 2018-08-17 14:43:19,347:DartRunner.py:2958:        SqlTests       NullTest     False         1 2018-08-17 14:42:59 2018-08-17 14:43:00
INFO: 2018-08-17 14:43:19,347:DartRunner.py:2968: Dart DashBoard URL: http://dartdashboard.labs.teradata.com/report?run_id=180817-116
INFO: 2018-08-17 14:43:19,347:DartRunner.py:1347: Dart DashBoard URL: http://dartdashboard.labs.teradata.com/report?run_id=180817-116
